.circleci/config.yml

```diff
@@ -27,11 +33,12 @@ executors:
   e2e-environment-executor:
     working_directory: ~/src
```

Comment:
Can we use `*working_dir` here as well?

Reply:
Yes. This configuration needs some love. The anchors are not used consequently etc. Do you want me to do it in this PR? :smiley:

Reply:
If you have more things in mind then a different PR, if you only want to fix that one, I'd just add it to this PR.

Reply:
Lets just do it. Always leave the code better than you found it.

Reply:
:heavy_check_mark: